src-preview/index.ts

```diff
@@ -130,37 +130,32 @@ document.addEventListener('click', (event) => {
   let node: any = event.target
   while (node) {
     if (node.tagName && node.tagName === 'A' && node.href) {
-      // if (node.getAttribute('href').startsWith('#')) {
```

Comment:
The Markdown extension is using a "simpler" implementation: https://github.com/microsoft/vscode/blob/main/extensions/markdown-language-features/preview-src/index.ts#L172-L205

Do we need to check if the href starts with `https://file+.vscode-resource.vscode-webview.net/`?